src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5117,6 +5117,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ExplicitReadOnlyAttr" xml:space="preserve">
     <value>Do not use 'System.Runtime.CompilerServices.IsReadOnlyAttribute'. This is reserved for compiler usage.</value>
   </data>
+  <data name="ERR_ExplicitIsByRefLikeAttr" xml:space="preserve">
+    <value>Do not use 'System.Runtime.CompilerServices.IsByRefLikeAttribute'. This is reserved for compiler usage.</value>
```

Comment:
I think declaring is fine. Same as with IsReadOnly - you can declare one, but cannot use it explicitly.

---
In reply to: [118793264](https://github.com/dotnet/roslyn/pull/19795#discussion_r118793264) [](ancestors = 118793264)